README.md

```diff
@@ -75,15 +158,13 @@ In this section you can find all commands supported by harbor:
 
 `harbor npm` will run npm command on node container and pass all additional arguments to npm.
 
-`harbor yarn` will run yarn command on node container and pass all additional arguments to yarn.
-
-`harbor gulp` will run gulp command on node container from node modules and pass all additional arguments to gulp.
+Support for `yarn` and `gulp` has been dropped.
 
-#### PGSQL container commands ####
+#### DB container commands ####
 
```

Comment:
tu tiez, mysql

Reply:
mariadb ma mysql command, takze tu je to spravne